### PR TITLE
changes to abstract

### LINKED ISSUE #528: Improvements to the abstract
The current abstract has a number of issues associated with it and it should be rewritten with the following changes:

1. Abstract should be 3-5 paragraphs
2. More business framing
3. Direct links to related references including: 
- DID Spec
- VC Spec
- Trace Vocab spec
4. references VC API should be moved to its own section "Relationship to VC-API"

### PR DESCRIPTION
**Pull request by mkhraisha**
closes #528 

Got kind of stuck writing this, would love any improvements or suggestions.

### CODE REVIEW

**Comment by brownoxford** [comment]
Looks like we just need bullet point 4, `references VC API should be moved to its own section "Relationship to VC-API"`

**Comment by mkhraisha** [comment]
That is tackled in issue #529

**Comment by mkhraisha** [comment]
The comment in this issue is about removing it

File: docs/spec/sections/abstract.html [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
+    Credentials](https://www.w3.org/TR/vc-data-model-2.0/), particularly as they relate to supply chain use cases.
+    Exchanged Verifiable Credentials often implement schemas defined in the [Traceability Vocab
+    Specification](https://w3c-ccg.github.io/traceability-vocab/).
+  </p>
+  <p>
+    Utilizing Verifiable Credentials to represent supply chain data enables interoperability between disparate actors,
+    and provides the ability to cryptographically verify the authenticity of the data.
+  </p>
+  <p>
+    This specification utilizes the [Decentralized Identifiers (DIDs) v1.0](https://www.w3.org/TR/did-core/)
```
Comment (TallTed) [inline docs/spec/sections/abstract.html:15]:
```suggestion
    This specification uses the [Decentralized Identifiers (DIDs) v1.0](https://www.w3.org/TR/did-core/)
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Minor

**Review by nissimsan — APPROVED** [review]

**Review by brownoxford — APPROVED** [review]

**Pull request merged**